core/templates/dev/head/dashboard/MyExplorations.js

```diff
@@ -31,9 +31,8 @@ oppia.controller('MyExplorations', [
       return RatingComputationService.computeAverageRating(ratings);
     };
 
-    $scope.showCreateExplorationModal = function() {
-      ExplorationCreationButtonService.showCreateExplorationModal(
-        CATEGORY_LIST);
+    $scope.createNewExploration = function() {
```

Comment:
Can't you just do: ``$scope.createNewExploration = ExplorationCreationService.createNewExploration``?